backend/interview_portal/src/main/java/by/interview/portal/controller/UserController.java

```diff
@@ -1,53 +1,72 @@
 package by.interview.portal.controller;
 
-import java.util.List;
-
+import by.interview.portal.domain.Role;
+import by.interview.portal.dto.FullUserInfoDTO;
+import by.interview.portal.dto.UserBaseInfoDTO;
+import by.interview.portal.dto.UserDTO;
+import by.interview.portal.facade.UserFacade;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-import by.interview.portal.domain.Role;
-import by.interview.portal.dto.FullUserInfoDTO;
-import by.interview.portal.dto.UserBaseInfoDTO;
-import by.interview.portal.facade.UserFacade;
+import java.util.List;
 
 @CrossOrigin
 @RestController
 @RequestMapping(value = "/users")
 public class UserController {
 
+    private static final Logger LOG = LogManager.getLogger(UserController.class);
```

Comment:
Probably, you can use Lombok's Slf4j annotation: https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html